src/main/java/com/synopsys/integration/detect/workflow/diagnostic/DiagnosticLogUtil.java

```diff
@@ -37,6 +35,6 @@
     }
 
     public static ch.qos.logback.classic.Logger getRootLogger() {
-        return getLogger(ch.qos.logback.classic.Logger.ROOT_LOGGER_NAME);
+        return getLogger(org.slf4j.Logger.ROOT_LOGGER_NAME);
```

Comment:
If this is returning a `ch.qos.logback.classic.Logger` why are we using the logger name from slf4j?

Reply:
the logger type we are using here implements slf4j, and that's where is gets ROOT_LOGGER_NAME from.  sonarCloud had issue with referencing a value from an implemented class that it does not override